src/shared/components/DurationSelector.tsx

```diff
@@ -17,13 +29,30 @@ interface Props {
   onSelectDuration: (duration: string) => any
   durations: DurationOption[]
   disabled?: boolean
+  status?: ComponentStatus
 }
 
+const durationUnits = ['day', 'month', 'year']
+
+// returns plural version of the unit if value is greater than 1
+// Examples: 1 day, 1 month, 12 days, 30 years
+const pluralizeUnitIfNeeded = (unit: string, value: number) => {
```

Comment:
Little function to pluralize units, when needed.

Reply:
Oh, my example was misleading. All I do in that function is take a magnitude and return the plural unit if needed. 

Like, `1 day`
example 2: `3 days`

I can see how that `1 day, 1 month, 12 days, 30 years` was misleading. 

Am I addressing your concern correctly?

Reply:
What I'm asking is: is it possible to have values of 0? If so, then you'll need to account for that when adding an `s` to the units. If it's not possible to have values of 0, then it's not a concern. I was trying to provide an example of when using 0 as a value might be reasonable